build/Helix/PrepareHelixPayload.ps1

```diff
@@ -26,10 +26,18 @@ cp build/Helix/ConvertWttLogToXUnit.cs $payloadDir
 
 cp build/Helix/ConvertWttLogToXUnit.ps1 $payloadDir
 
-cp $Platform-$Configuration-taeftest/ProjectReunionHelix.dll $payloadDir
+cp BuildOutput/$Configuration/$Platform/BaseUnitTests/BaseUnitTests.dll $payloadDir
```

Comment:
Do you feel OK about hard-wiring this here? (question only, not blocking comment)

Reply:
I am okay with it for now. For example, WinUI3 uses another task to gather ALL of the test binaries and bin place them in one place along with the rest of the payload for the helix machine.  We really only have the one test binary right now but this is open to include others.  Regardless of which way we chose to do this we will have to modify some task/powershell script that gathers everything together into one directory before sending it over to the helix machine.  I think this script is self descriptive and easy to understand what it is doing.  It's also simple enough to add to it when we need to add more tests and more stuff will have to be included with the payload.  

I think the downside is if we update the TAEF redist version or move anything around this will break our testing pipeline. But we will be notified about that before a PR is checked in.